python/grass/temporal/core.py

```diff
@@ -470,7 +470,7 @@ def stop_subprocesses():
 atexit.register(stop_subprocesses)
 
 
-def get_available_temporal_mapsets():
+def get_available_temporal_mapsets(only_current_mapset=False):
     """Return a list of of mapset names with temporal database driver and names
     that are accessible from the current mapset.
```

Comment:
The parameter name is quite clear, but it goes against what the current documentation say, please modify to include the parameter.

Reply:
Documentation has been updated.